pkg/app/piped/apistore/eventstore/store.go

```diff
@@ -62,6 +62,10 @@ type store struct {
 
 const (
 	defaultSyncInterval = time.Minute
+	// A week.
+	durationToFetchAtStartUp = 168 * time.Hour
```

Comment:
```suggestion
	durationToFetchAtStartUp = 7 * 24 * time.Hour
```

Reply:
Btw, do we really need to load that far?

Reply:
Nice point. I'm still debating it.

For example, if Piped has not been running for some reason for a couple of days, the latest events will not be handled forever, which may confuse users.

Reply:
But in that case, you can just issue the event again, so 24 hours might be fine.